.changeset/fluffy-rockets-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphprotocol/graph-cli': minor
```

Comment:
should be `major` because of breaking changes

Reply:
https://github.com/graphprotocol/graph-tooling/pull/1500#discussion_r1392713797